ouroboros-consensus/src/Ouroboros/Consensus/MiniProtocol/BlockFetch/Server.hs

```diff
@@ -129,7 +131,5 @@ blockFetchServer _tracer chainDB _version registry = senderSide
 -------------------------------------------------------------------------------}
 
 -- | Events traced by the Block Fetch Server.
-data TraceBlockFetchServerEvent blk
-   -- TODO no events yet. Tracing the messages send/received over the network
-   -- might be all we need?
+newtype TraceBlockFetchServerEvent blk = TraceBlockFetchServerBlockCount Int
```

Comment:
Please remove the `Int`, add a docstring, use the following style, and use `data`. I know it has one constructor, but that's just a coincidence.

```
data TraceBlockFetchServerEvent blk =
    -- | The server sent a block to the peer.
    TraceBlockFetchServerSendBlock
```